src/params.lean

```diff
@@ -28,16 +46,22 @@ open params
 
 variables [params.{u}]
 
+/-- To allow Lean's type checker to see that the ordering `Λr` is a well-ordering without having to
+explicitly write `Λwf` everywhere, we declare it as an instance.
+We can then deduce from the well-ordering that `Λ` is linearly ordered. -/
```

Comment:
Do you really want this to be a docstring about `is_well_order Λ Λr` but not `linear_order Λ`?

Reply:
I'll fix this locally and commit soon.